xml/security_cryptofs.xml

```diff
@@ -264,9 +273,150 @@
   </para>
   <para>
     Keep in mind that the encrypted file can be only decrypted using the same
-    key that was used for encryption. If you want to share an encrypted file
+    key that was used for encryption. To share an encrypted file
     with another person, you have to use that person's public key to encrypt
     the file.
   </para>
  </sect1>
+ <sect1 xml:id="sec-security-cryptofs-rage">
+  <title>Encrypting files with Rage</title>
+  <para>
+   Rage is a secure file encryption software to encrypt files. It has keys that
+   are easy to exchange with other people, and has secure defaults to prevent
+   accidental misuse or leaks of sensitive data. We recommend Rage to encrypt
+   files.
+  </para>
+  <para>
+   The recipient must first generate a key pair to encrypt a file with Rage:
+  </para>
+  <screen>&prompt.user;rage-keygen -o ~/rage.key 2 ~/rage.pub</screen>
+  <para>
+   Two files are created; <filename>rage.pub</filename> and <filename>rage.key</filename>.
+  </para>
+  <example>
+   <title><filename>rage.pub</filename> example</title>
+   <screen>&prompt.user;cat rage.pub
+Public key: age17e4g67cs07jk3lmylyq6gduv26uf7tz7nm9jrsaxn8xxx9uc9amsdg4a5e</screen>
+  </example>
+  <example>
+   <title><filename>rage.key</filename> example</title>
+   <screen>&prompt.user;cat file.key
+# created: 2023-05-30T16:29:20+05:30
+# public key: age17e4g67cs07jk3lmylyq6gduv26uf7tz7nm9jrsaxn8xxx9uc9amsdg4a5e</screen>
+  </example>
+  <important>
+   <title>Never share rage.key</title>
+   <para>
+    <emphasis>rage.key</emphasis> is a private key and should be kept confidential.
+   </para>
+  </important>
```

Comment:
I don't think that's needed, because then there is a whole discussion of "do you keep the private key long term or not" and "once you decrypt a file how to do you make sure it's content never leaks" etc. Rage protects the file from point A to B, not to protect a file locally for all of time. There also a whole discussion there about key longevity too. As well, once you decrypt the file then you have the decrypted file ...